src/lib/tls/tls_session_manager_memory.cpp

```diff
@@ -107,8 +107,7 @@ size_t Session_Manager_In_Memory::remove_internal(const Session_Handle& handle)
          std::erase_if(m_sessions, [&](const auto& item)
             {
             const auto& [_, session_and_handle] = item;
-            const auto& [__, this_handle] = session_and_handle;
-
+            const auto& [_unused, this_handle] = session_and_handle;
```

Comment:
Maybe `_unused1` and `_unused2` for consistency? I found double-underscore was kind-of good enough to work around the 'repeated identifier' issue. But if clang-tidy doesn't like that, so be it.

Reply:
Yeah that's fine. For context the complaint here is because C++ reserves all identifiers containing `__` for the implementation.